tests/test_db_function.py

```diff
@@ -310,10 +310,18 @@ def test_get_data(seed_application_records):
             }
         },
     )
-
-    assert data[0]["Charity number "] == "Test"
-    assert data[0]["Do you need to do any further feasibility work?"] is False
-    assert data[0]["Project name"] == "Save the humble pub in Bangor"
-    assert data[0]["Risks to your project (document upload)"] == "sample1.doc"
-    assert data[0]["Score"] == 5
-    assert data[0]["Score Justification"] == "great"
+    # TODO add test data for cy_list
```

Comment:
is there a blocker for the welsh data here?

Reply:
Not really a blocker, more off would be nice